var/spack/repos/builtin/packages/genesis/package.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Genesis(AutotoolsPackage, CudaPackage):
+    """GENESIS is a Molecular dynamics and modeling software
+    for bimolecular systems such as proteins, lipids, glycans,
+    and their complexes.
+    """
+
+    homepage = "https://www.r-ccs.riken.jp/labs/cbrt/"
+    url = "https://www.r-ccs.riken.jp/labs/cbrt/wp-content/uploads/2020/09/genesis-1.5.1.tar.bz2"
+    git = "https://github.com/genesis-release-r-ccs/genesis-2.0.git"
+
+    version("2.0b", branch="master")
+    version(
+        "1.5.1",
+        sha256="62a453a573c36779484b4ffed2dfa56ea03dfe1308d631b33ef03f733259b3ac",
+        url="https://www.r-ccs.riken.jp/labs/cbrt/wp-content/uploads/2020/09/genesis-1.5.1.tar.bz2",
+    )
+
+    resource(
+        when="@1.5.1",
+        name="user_guide",
+        url="https://www.r-ccs.riken.jp/labs/cbrt/wp-content/uploads/2019/10/GENESIS-1.4.0.pdf",
+        sha256="da2c3f8bfa1e93adb992d3cfce09fb45d8d447a94f9a4f884ac834ea7279b9c7",
+        expand=False,
+        placement="doc",
+    )
+
+    variant("openmp", default=True, description="Enable OpenMP.")
+    variant("single", default=False, description="Enable single precision.")
+    variant("hmdisk", default=False, description="Enable huge molecule on hard disk.")
+
+    conflicts("%apple-clang", when="+openmp")
+
+    depends_on("autoconf", type="build", when="@1.5.1 %fj")
+    depends_on("autoconf", type="build", when="@2.0b")
+    depends_on("automake", type="build", when="@1.5.1 %fj")
+    depends_on("automake", type="build", when="@2.0b")
+    depends_on("libtool", type="build", when="@1.5.1 %fj")
+    depends_on("libtool", type="build", when="@2.0b")
+    depends_on("m4", type="build", when="@1.5.1 %fj")
+    depends_on("m4", type="build", when="@2.0b")
+
+    depends_on("mpi", type=("build", "run"))
+    depends_on("lapack")
+    depends_on("python@2.7.18", type=("build", "run"), when="@2.0b")
```

Comment:
This should use a range of supported Python versions.

Reply:
I set a range of supported Python versions.